app/src/test/java/com/x8bit/bitwarden/data/vault/manager/VaultLockManagerTest.kt

```diff
@@ -210,7 +215,6 @@ class VaultLockManagerTest {
                 VaultTimeout.FourHours,
                 is VaultTimeout.Custom,
                 -> {
-                    assertTrue(vaultLockManager.isVaultUnlocked(USER_ID))
```

Comment:
These assertions were removed as they are not relevant since locking the vault takes place after call to soft logout in new flow and these are decoupled and can now be tested in isolation